src/test/java/io/rocketpartners/cloud/action/rest/TestRestGetActions.java

```diff
@@ -116,4 +119,108 @@ public void testPagination01() throws Exception
       assertEquals(830, total);
    }
 
+   @Test
+   public void testEw01() throws Exception
+   {
+      Service service = service();
+      Response res = null;
+      res = service.get(url("orders?ew(shipname,Chevalier)"));
+      ArrayNode data = res.data();
+      boolean assertion = true;
```

Comment:
Functionally good.  Would prefer this...

assertTrue(data.size() > 0);
for (Object o : data)
{
  assertTrue(((ObjectNode)o).getString("shipname").endsWith("Chevalier")));
}